crates/holochain/tests/tests/test_utils/mod.rs

```diff
@@ -257,18 +257,18 @@ pub async fn attach_app_interface(client: &WebsocketSender, port: Option<u16>) -
     }
 }
 
-pub async fn retry_admin_interface(
+pub async fn retry_websocket_client_by_port(
```

Comment:
Existing test utils helper that wasn't actually used anywhere. Updated to make it useful.